frontend/src/pages/ActivityReport/Pages/goalsObjectives.js

```diff
@@ -66,7 +66,17 @@ const GoalsObjectives = ({ reportId }) => {
       try {
         if (isRecipientReport && hasGrants) {
           const fetchedGoals = await getGoals(grantIds);
-          const formattedGoals = fetchedGoals.map((g) => ({ ...g, grantIds }));
+          const formattedGoals = fetchedGoals.map((g) => {
+            // if the goal is on an "old" grant, we should
+            // treat it like a new goal for now
+            let isNew = false;
```

Comment:
I think this code breaks the retrieval of getting the objectives for existing goals.

see GoalForm.js line 98 "if (!goal.isNew && goal.goalIds) " the first part of this IF is never true for a goal with objectives. So it never attempts to retrieve the goal objectives from the db.

Reply:
Let me know once this is fixed and I will continue testing.

Reply:
This issue has been addressed